### PR TITLE
Farman development

### PR DESCRIPTION
**Pull request by farmanahmed2007**

### CODE REVIEW

File: src/components/build/mainPage/mainPage.scss [hunk 1]
```diff
@@ -85,22 +85,6 @@
...
-
-  .zoom-item:hover {
-    transform: scale(1.4);
-
-    .item-description {
-      color: white;
-      font-family: $font-family-bold;
-    }
-  }
-
```
Comment (ivanromakh) [inline src/components/build/mainPage/mainPage.scss:103]: @farmanahmed2007 Why did you move this class to global.scss if it is used only in main page?

Comment (farmanahmed2007) [reply src/components/build/mainPage/mainPage.scss:103]: I can use somewhere else and generalize the scale effect
